frontends/web/src/locales/en/app.json

```diff
@@ -525,7 +526,7 @@
         "title": "Welcome to your one stop shop for buying {{name}}"
       },
       "next": "Next",
-      "selectPlaceholder": "Select a coin",
+      "selectLabel": "Select an account",
```

Comment:
Already updated in locize

![Screen Shot 2021-05-21 at 12 49 02 PM](https://user-images.githubusercontent.com/546900/119126567-7725da80-ba33-11eb-8dff-d1d2efbb9290.png)

Reply:
please don't edit text in the json file, only in locize: https://github.com/digitalbitbox/bitbox-wallet-app/blob/master/docs/i18n.md

deleting the `selectPlaceholder` is good.